met-web/src/components/engagement/form/EngagementFormTabs/EngagementContent/ContentTabs.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { useContext, useState } from 'react';
 import { Box, Button, IconButton, MenuItem, Menu, Skeleton, Grid, Divider } from '@mui/material';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTrash, faPenToSquare, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faTrash, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faSquarePen } from '@fortawesome/pro-regular-svg-icons';
```

Comment:
Could we import faTrash and faPlus from the pro tier as well? or do they only exist in the free package

Reply:
this change I did to just to confirm whether the pro-integration is working. We may have another ticket to change whole icons to pro or in the future we can both